zoo/repos/tasks.py

```diff
@@ -154,3 +158,24 @@ def get_zoo_file_content(proj: Dict) -> str:
     return provider.get_file_content(
         proj["id"], settings.ZOO_YAML_FILE, settings.ZOO_YAML_DEFAULT_REF
     )
+
+
+def sync_enviroments_from_gitlab(repo: Repository):
+    RepositoryEnvironment.objects.filter(repository_id=repo.id).delete()
```

Comment:
Why not using `objects.get_or_create` instead?

Reply:
Well this task needs to sync envs from gitlab. What if some env was deleted on gitlab. I am not sure that  get_or_create would remove deleted env from db.